src/qt/peertablemodel.h

```diff
@@ -74,22 +62,25 @@ class PeerTableModel : public QAbstractTableModel
 
     /** @name Methods overridden from QAbstractTableModel
         @{*/
-    int rowCount(const QModelIndex &parent) const override;
-    int columnCount(const QModelIndex &parent) const override;
-    QVariant data(const QModelIndex &index, int role) const override;
-    QVariant headerData(int section, Qt::Orientation orientation, int role) const override;
-    QModelIndex index(int row, int column, const QModelIndex &parent) const override;
+    int rowCount(const QModelIndex& parent = QModelIndex()) const override;
+    int columnCount(const QModelIndex& parent = QModelIndex()) const override;
+    QVariant data(const QModelIndex& index, int role = Qt::DisplayRole) const override;
+    QVariant headerData(int section, Qt::Orientation orientation, int role = Qt::DisplayRole) const override;
+    QModelIndex index(int row, int column, const QModelIndex& parent = QModelIndex()) const override;
     Qt::ItemFlags flags(const QModelIndex &index) const override;
-    void sort(int column, Qt::SortOrder order) override;
     /*@}*/
 
 public Q_SLOTS:
     void refresh();
 
+Q_SIGNALS:
+    void changed();
+
 private:
+    //! Internal peer data structure.
+    QList<CNodeCombinedStats> m_peers_data{};
```

Comment:
nit: `{}` seems redundant, it's not fundamental type like int, pointer or bool to need this kind of explicit default initialization, constructor will be invoked regardless.

Reply:
Yes, it is redundant. Leaving as is to be explicit.